test/jdk/java/lang/Process/ProcessCloseTest.java

```diff
@@ -0,0 +1,662 @@
+        PROCESS_DESTROY(ProcessCommand::processDestroy),
+        PROCESS_FORCE_OUT_CLOSE_EXCEPTION(ProcessCommand::processForceOutCloseException),
+        PROCESS_FORCE_IN_CLOSE_EXCEPTION(ProcessCommand::processForceInCloseException),
+        PROCESS_FORCE_ERROR_CLOSE_EXCEPTION(ProcessCommand::processForceErrorCloseException),
+        WRITER_WRITE(ProcessCommand::writerWrite),
+        WRITER_CLOSE(ProcessCommand::writerClose),
+        STDOUT_PRINT_ALL_LINES(ProcessCommand::stdoutPrintAllLines),
+        STDERR_PRINT_ALL_LINES(ProcessCommand::stderrPrintAllLines),
+        STDOUT_WRITE(ProcessCommand::stdoutWrite),
+        STDOUT_CLOSE(ProcessCommand::stdoutClose),
+        STDOUT_EXPECT_POLO(ProcessCommand::stdoutExpectPolo),
+        STDERR_EXPECT_POLO(ProcessCommand::stderrExpectPolo),
+        STDOUT_EXPECT_EMPTY(ProcessCommand::stdoutExpectEmpty),
+        STDERR_EXPECT_EMPTY(ProcessCommand::stderrExpectEmpty),
+        ;
+        private final Consumer<Process> command;
+
+        ProcessCommand(Consumer<Process> command) {
+            this.command = command;
+        }
+
+        private static void stdoutPrintAllLines(Process p) {
+            try {
+                var lines = p.inputReader().readAllLines();
+                Assertions.assertNotEquals(0, lines.size(), "stdout should not be empty");
+                System.err.printf("        %d lines\n", lines.size());
+                System.err.println(lines.toString().indent(8));
+            } catch (IOException ioe) {
+                throw new UncheckedIOException(ioe);
+            }
+        }
+
+        private static void stderrPrintAllLines(Process p) {
+            try {
+                var lines = p.errorReader().readAllLines();
+                Assertions.assertNotEquals(0, lines.size(), "stderr should not be empty");
+                System.err.printf("        %d lines\n", lines.size());
+                System.err.println(lines.toString().indent(8));
+            } catch (IOException ioe) {
+                throw new UncheckedIOException(ioe);
+            }
+        }
+
+        private static void writerWrite(Process p) {
+            try {
+                p.outputWriter().write("Now is the time.");
+            } catch (IOException ioe) {
+                throw new UncheckedIOException(ioe);
+            }
+        }
+
+        private static void writerClose(Process p) {
+            try {
+                p.outputWriter().close();
+            } catch (IOException ioe) {
+                throw new UncheckedIOException(ioe);
+            }
+        }
+
+        private static void stdoutExpectPolo(Process p) {
+            String line = readLine(p.getInputStream());
+            Assertions.assertEquals("Polo", line, "Stdout Expected Polo");
+        }
+
+        private static void stderrExpectPolo(Process p) {
+            String line = readLine(p.getErrorStream());
+            Assertions.assertEquals("Polo", line, "Stderr Expected Empty");        }
```

Comment:
This looks like a typo in the error message. I think it should be "Stderr Expected Polo".